src/win/fs.c

```diff
@@ -113,9 +113,45 @@ const WCHAR LONG_PATH_PREFIX_LEN = 4;
 const WCHAR UNC_PATH_PREFIX[] = L"\\\\?\\UNC\\";
 const WCHAR UNC_PATH_PREFIX_LEN = 8;
 
+static DWORD uv__file_symlink_flags;
+
+static int uv__usermode_symlink_available(void) {
+  HKEY key;
+  DWORD value, value_size;
+  OSVERSIONINFOW version_information;
+  int r;
+
+  if (!pRtlGetVersion) {
+    return 0;
+  }
+  pRtlGetVersion(&version_information);
+  /* Usermode symlinks are only available on Win10 with Creators Update... */
+  if (version_information.dwMajorVersion < 10 ||
```

Comment:
This is the classic version check mistake. What happens (who knows!) when Win11 build 1000 comes out?